src/components/Layout/Layout.css

```diff
@@ -0,0 +1,61 @@
+body {
+    background-color: black;
+    color: white;
+    font-family: 'Jura', sans-serif;
+    font-size: 1vw;
+}
+
```

Comment:
I'd suggest styling the <a> element in order to remove the default blue/purple color, just to match the app's design:

```
a {
    color: white;
}
```

<img width="303" alt="image" src="https://github.com/Code-the-Dream-School/ffprac-team1-front/assets/42615340/d011f3a4-964b-4294-b840-0580c59cb2ad">